src/components/ScaledImage/ScaledImage.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { SyntheticEvent, EventHandler, ReactElement, ImgHTMLAttributes } from 'react';
+import { ItemImage } from 'components/types';
+
+interface Props extends ImgHTMLAttributes<HTMLImageElement> {
+  image?: ItemImage;
+  fallbackUrl?: string;
+}
+
+export function tmhImageUrl(size: number, imageSrc: string) {
+  let baseUrl: string;
+  if (window.location.hostname === 'localhost') {
+    baseUrl = 'https://localhost:3006';
+  } else if (window.location.hostname.includes('beta')) {
+    baseUrl = 'https://beta.themeetinghouse.com/cache/' + size;
+  } else {
+    baseUrl = 'https://www.themeetinghouse.com/cache/' + size;
+  }
+  return `${baseUrl}${imageSrc} ${size}w`;
+}
+
+export function fallbackToImage(fallbackUrl: string | undefined): EventHandler<SyntheticEvent<HTMLImageElement>> | undefined {
+  if (!fallbackUrl) {
+    return
+  }
+  return function (event: SyntheticEvent<HTMLImageElement>) {
+    if (!event.currentTarget.src.endsWith(fallbackUrl)) {
+      event.currentTarget.src = fallbackUrl;
+      event.currentTarget.srcset = '';
+    }
+  };
+}
+
+export default function ScaledImage(props: Props): ReactElement<Props> | null {
+  if (!props?.image?.src) {
+    return null;
+  }
+
+  const srcSet = [
```

Comment:
@nikklassen  Might be nice going forward to have ability to define/pick from a limited number of srcSet's...hero images are much different than staff photo's. But there aren't that many variations, maybe just two right now...

Reply:
Sure, makes sense. I'll see which way is cleaner.